docs/framework/install/toc.md

```diff
@@ -1,4 +1,12 @@
 # [Installation Guide](index.md)
-## [Installing the .NET Framework 3.5 on Windows 8 and later versions](net-framework-3-5-on-windows-8-plus.md)
-## [Running .NET Framework 1.1 Apps on Windows 8 and later versions](run-net-framework-1-1-apps.md)
-## [Troubleshooting](troubleshoot-blocked-installations-and-uninstallations.md)
+## [Running .NET Framework 1.1 Apps on Windows 8, Windows 8.1, or Windows 10](run-net-framework-1-1-apps.md)
+## [Install the .NET Framework 3.5 on Windows 10, Windows 8.1, and Windows 8](dotnet-35-windows-10.md)
+## [Troubleshooting the installation of the .NET Framework 3.5 on Windows 8, Windows 8.1, and Windows 10](net-framework-3-5-on-windows-8-plus.md)
+## [Installing the .NET Framework](guide-for-developers.md)
```

Comment:
@mairaw Rename *Installing the .NET Framework* (guide-for-developers.md) :point_right: *Install the .NET Framework for developers*?